src/libecalc/core/models/turbine.py

```diff
@@ -40,6 +40,7 @@ def max_power(self) -> Optional[float]:
         )
 
     def evaluate(self, load: NDArray[np.float64], fuel_lower_heating_value: float = 0) -> TurbineResult:
+        # Why do we divide by the adjustment factor here, and not pure linear adjustment?
```

Comment:
probably incorrect? I guess one has thought opposite of what is correct - similarily to power loss factor perhaps? but it wasnt implemented before right, so it didnt have any effect of dividing by 1?

Reply:
That is true. It was defaulted to 1

Reply:
@olelod do you recall if this is intentional or not? (dividing instead of linear adjustment for turbine)

Reply:
From what I understand, there is a need to adjust/calibrate at turbine level: It is a difference between polytropic efficiency in the compressor, and loss in the turbine. Hence, it is my understanding that it should be possible to adjust them separately.